CWE/events/italy.txt

```diff
@@ -1899,6 +1903,237 @@ country_event = {
+		}
+	}
+
+	mean_time_to_happen = { months = 2 }
+	
+	option = {
+		name = "'Organic' Centre-Right (DC-PLI-MSI PDI)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = 0.02 value = conservative }
+		}
+		any_pop = { 
+			limit = { is_primary_culture = yes OR = { strata = rich strata = middle } }
+			ideology = { factor = 0.04 value = nationalist } 
+		}
+		any_pop = { 
+		    limit = { 
+		        OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_763 region = SAR_765 } 
+			}
+			ideology = { factor = 0.08 value = traditionalist } 
+		}  
+		set_country_flag = 1963_ITA_Organic_Centre_Right
+		ai_chance = { factor = 0.8 }
+	}
+	
+	option = {
+		name = "'Organic' Centre-Left (DC-PSDI-PSI-PRI)"
+		ruling_party_ideology = big_tent
+		any_pop = { 
+			ideology = { factor = 0.07 value = big_tent }
+			ideology = { factor = 0.02 value = socialist }
+		}
+		any_pop = { 
+			limit = { OR = { strata = middle strata = poor } }
+			ideology = { factor = 0.02 value = communist }
+		}
+		
+		set_country_flag = 1963_ITA_Organic_Centre_Left
+		ai_chance = { factor = 0.2 }
+	}
+
+}
+
+#Death of Togliatti
+country_event = {
+  id = 8020332
+  title = "The Death of Togliatti"
+  desc = EVTDESC8020332
+  picture = "togliatti_funeral"
+  
+  fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1964
+		NOT = { year = 1990 }
+	}
+
+	mean_time_to_happen = { months = 8 }
+
+	option = {
+		name = "Farewell Palmiro! Farewell!"
+		any_pop = { 
+			limit = { strata = poor }
+			ideology = { factor = 0.04 value = communist } 
+			ideology = { factor = 0.02 value = communist_social } 
+		}
+		any_pop = { 
+            scaled_consciousness = { factor = 1 ideology = communist }
+			scaled_militancy = { factor = 2 ideology = communist }
+			scaled_consciousness = { factor = 1 ideology = communist_social }
+			scaled_militancy = { factor = 2 ideology = communist_social }
+		}
+	}
+
+}
+
+#1968 Elections Historical
+country_event = {
+	id = 8020333
+	title = "$YEAR$ Italian General Election"
+	desc = EVTDESC8020333
+	picture = "ITA_1968_elections"
+	
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1968
+		NOT = { year = 1973 }
+		
+		government = democracy
+		war = no
+		is_vassal = no
+		
+		OR = { has_country_flag = 1963_ITA_Organic_Centre_Right has_country_flag = 1963_ITA_Organic_Centre_Left has_country_flag = 1963_ITA_Centrism }
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "'Organic' Centre-Left (DC-PSDI-PSI-PRI)"
+		ruling_party_ideology = big_tent
+		any_pop = { 
+			ideology = { factor = 0.01 value = big_tent }
+		}
+		any_pop = { 
+			limit = { OR = { strata = middle strata = poor } }
+			ideology = { factor = 0.03 value = communist }
+		}
+		any_pop = { 
+			limit = { OR = { strata = middle strata = rich } }
+			ideology = { factor = 0.02 value = conservative }
+		}
+		
+		set_country_flag = 1968_ITA_Organic_Centre_Left
+	}
+
+}
+
```

Comment:
I know every historical event needs an ahistorical option, I will very soon add that but through events that will be triggered by this event

Reply:
I’m not a fan of a visual novel style of creating events (one option event leading to another) so I prefer if a direct option to avoid changing ruling party ideology is made here so as not to completely box in the player into accepting a centrist party.

Reply:
The Problem is that OTL there was no way another way a majority could be formed, I tried to look into any other possible coalition but as much as I digged I couldn't find anything that wasn't just the same centrist coalition just with less parties

Reply:
So I really wouldn't know what else to put there

Reply:
I might have found a way but would it be a problem if both options lead to a big_tent government and one of those two can later become of another ideology? alternatively if that's an issue I can make the path that can change ideology later not change ideology during the elections event

Reply:
I see, maybe the other option can just be to call another election (on the premise that a coalition was unable to be formed at all for some reason) then add some militancy here and there plus possibly some ideology support for the socialists. (must also add a election = no check if this is done).

Also, I would a like a trigger check for ITA being run by a big_tent gov for this event as I think the reasons why the election was so close was because the centrist gov faced difficulties so the effects would not make sense for any other party which is in power.

Reply:
Yeah another elections soon after is what I was thinking of doing. I don't know how to feel about the big_tent check as that would either mean that if you go with the other choice you don't get elections for that year or I should create an alternate history even choice, which I could do but I'd prefer to do after having finished historical path, so the current solution with the no ideology change could be kept as a compromise until I get on alternate history if it's alright with you

Reply:
It’s fine not to have an election event for that year if an AH path has been taken because of either an earlier event option or because some generic system intervened. The events should be scripted as tightly as possible because it’s going to be annoying to the player if an event pops up out of context, even more so if they have no option to avoid it.